sig-security/annual-report-2021.md

```diff
@@ -0,0 +1,81 @@
+    *   We measure our reviewer bandwidth by noting whether everything brought to SIG Security’s attention is getting addressed in a timely fashion, and the feelings of the reviewers about their workload.
+    *   Much of this review is currently done by the SIG chairs, and it is starting to become a bottleneck. We are looking to improve our review bandwidth by bringing review of triaged issues to the regular SIG meetings for more open and inclusive participation. 
+*   **Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+    *   We are onboarding and growing contributors continuously through our processes and culture. We actively practice behaviors of inclusion to help new and growing contributors feel more confident to participate. For example, we list appropriate pronouns in our meeting attendance list, and before changing subjects we pause to ask for additional comments.
+    *   In addition to participating in SIG discussions, new contributors can also begin by assisting the subprojects with their tasks. Subproject members guide new contributors to where assistance is needed.
+    *   Our open environment creates space for interested contributors to become leaders [by sharing their ideas and energy](https://twitter.com/PuDiJoglekar/status/1380205701994147845). If someone brings an idea that other members are excited about and want to help work on, [those members can organize a subproject together](https://twitter.com/coffeeartgirl/status/1331046904306622465), and we can help them grow. 
+    *   The chairs help grow emerging leaders by keeping in close contact with them and providing 1:1 mentorship, [encouragement](https://twitter.com/coffeeartgirl/status/1337078959020912642), and concrete help leveling up skills such as improving communication, leading more inclusive meetings, and presenting at conferences.
+*   **What programs do you participate in for new contributors?**
+    *   Our broadly welcoming culture is a good opportunity for security-minded Kubernetes users who want to get involved in the upstream Kubernetes community. We plan to extend this further by participating in formal “new contributor” programs in the future.
+*   **Does the group have contributors from multiple companies/affiliations?**
+    *   Yes
+    *   **Can end users/companies contribute in some way that they currently are not?**
+        *   End users are currently contributing at all levels. Our contributors represent both end user organizations and Kubernetes vendors. Both SIG Security co-chairs work at companies that are Kubernetes end users.
```

Comment:
if you had a microphone to promote sig-security to all end users, what would you say you need the most help with or how you'd like them to interact with the group?